src/Module/Settings/Display.php

```diff
@@ -297,19 +330,19 @@ private function getAvailableTimelines(int $uid, bool $only_network = false): ar
 		$timelines = [];
 
 		foreach ($this->timeline->getNetworkFeeds('') as $channel) {
-			$timelines[$channel->code] = $this->t('%s: %s', $channel->label, $channel->description);
+			$timelines[$channel->code] = $channel;
 		}
 
 		if ($only_network) {
 			return $timelines;
 		}
 
 		foreach ($this->timeline->getChannelsForUser($uid) as $channel) {
-			$timelines[$channel->code] = $this->t('%s: %s', $channel->label, $channel->description);
+			$timelines[$channel->code] = $channel;
 		}
 
 		foreach ($this->timeline->getCommunities(true) as $community) {
-			$timelines[$community->code] = $this->t('%s: %s', $community->label, $community->description);
+			$timelines[$community->code] = $community;
```

Comment:
No need for this anymore.
```suggestion
			$timelines[] = $community;
```

Reply:
Done